### DISCUSSION TITLE
Ability to remove empty gap at top?

### DISCUSSION THREAD

**Discussion by Jaz1140** [dwainscheeren/dwains-lovelace-dashboard#717] [Ideas]
I have dwains dashboard running on 2 Samsung S7 FE tablets in my home, there is a large gap at the top that doesn't seem to serve a purpose, how can I remove it, or move my tiles and menu up?

I feel like I could see a 3 more rooms without having to scroll if it all moved up.

![PXL_20230712_030048255~2](https://github.com/dwainscheeren/dwains-lovelace-dashboard/assets/127198952/13475d6d-6cbf-4633-b142-44ee6fc5e14d)

**Comment by cnewman402** [comment]
This is in the /config/custom_components/dwains_dashboard/js/dwains-dashboard.js
Lines 150-157 or so.
It uses @media with padding of env, which may be used in other areas for the padding to draw right throughout the coding. 
More info: https://developer.mozilla.org/en-US/docs/Web/CSS/env

If you want to change this, you should make sure you have a backup, then try changing your media sizes -  i.e:
@media only screen and (max-width: 768px) {

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
